resources/views/about/index.blade.php

```diff
@@ -0,0 +1,229 @@
+                <td class='text-right'>{{ $stat_diskio }}</td>
+            </tr>
+            <tr>
+                <td><i class='fa fa-fw fa-cube fa-lg icon-theme' aria-hidden='true'></i> <b>@lang('HR-MIB')</b></td>
+                <td class='text-right'>{{ $stat_hrdev }}</td>
+                <td><i class='fa fa-fw fa-cube fa-lg icon-theme' aria-hidden='true'></i> <b>@lang('Entity-MIB')</b></td>
+                <td class='text-right'>{{ $stat_entphys }}</td>
+            </tr>
+            <tr>
+                <td><i class='fa fa-fw fa-clone fa-lg icon-theme' aria-hidden='true'></i> <b>@lang('Syslog Entries')</b></td>
+                <td class='text-right'>{{ $stat_syslog }}</td>
+                <td><i class='fa fa-fw fa-bookmark fa-lg icon-theme' aria-hidden='true'></i> <b>@lang('Eventlog Entries')</b></td>
+                <td class='text-right'>{{ $stat_events }}</td>
+            </tr>
+            <tr>
+                <td><i class='fa fa-fw fa-dashboard fa-lg icon-theme' aria-hidden='true'></i> <b>@lang('Sensors')</b></td>
+                <td class='text-right'>{{ $stat_sensors }}</td>
+                <td><i class='fa fa-fw fa-wifi fa-lg icon-theme' aria-hidden='true'></i> <b>@lang('Wireless Sensors')</b></td>
+                <td class='text-right'>{{ $stat_wireless }}</td>
+            </tr>
+            <tr>
+                <td><i class='fa fa-fw fa-print fa-lg icon-theme' aria-hidden='true'></i> <b>@lang('Toner')</b></td>
+                <td class='text-right'>{{ $stat_toner }}</td>
+                <td></td>
+                <td></td>
+            </tr>
+        </table>
+
+        <h3>@lang('License')</h3>
+        <pre>
+Copyright (C) 2013-{{ date('Y') }} {{ $project_name }} Contributors
+Copyright (C) 2006-2012 Adam Armstrong
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <a href="http://www.gnu.org/licenses/">http://www.gnu.org/licenses/</a>.</pre>
+
+        </div>
+    </div>
+</div>
+@endsection
+
+@section('javascript')
+<script>
+    $(document).ready(function () {
+        $("[name='statistics']").bootstrapSwitch('offColor','danger','size','mini');
+        $('input[name="statistics"]').on('switchChange.bootstrapSwitch',  function(event, state) {
+            event.preventDefault();
+            $.ajax({
+                type: 'POST',
+                url: '{{ url('/ajax/form') }}',
```

Comment:
Just change this to `url: 'ajax_form.php',`
Looks like it was broken on accident last year...